### PR TITLE
Feature/ls remote work blog

### PR DESCRIPTION
**Pull request by Lydias303**
Part 1 of 2 blog about working remotely effectively.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/wombat-security/wombat-security.github.io/pull/31" target="_blank">ReviewNinja

File: _drafts/2015-10-23-ls-remote-working.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+tags: development
+author_github:
+- Lydias303
+date: 2015-10-23 10:31:37
+comments: true
+banner_image: remote-work.jpg
+---
+###How to work effectively remotely:: Part 1 Environment and Productivity
+
+I recently attended a handful of talks and workshops at Denver StartUp week. One of which was about working effectively remotely. Four leaders in the remote industry were there to share their experiences and answer questions. Here are a few tips, tricks and pieces of advice I picked up there. As there were for individuals with varying pieces of advice I have grouped them into a few sections. We will begin this two part blog by discussing some of the most challenging aspects of remote work. Making a separate working environment and finding how you are most productive.
```
Comment (laurawhalin) [inline _drafts/2015-10-23-ls-remote-working.md]: I'd change the period after "Startup week" to a comma
small typo "As there were for individuals" -> change for to four
I'd change the period after "challenging aspects of remote work" to a colon:

File: _drafts/2015-10-23-ls-remote-working.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+###How to work effectively remotely:: Part 1 Environment and Productivity
+
+I recently attended a handful of talks and workshops at Denver StartUp week. One of which was about working effectively remotely. Four leaders in the remote industry were there to share their experiences and answer questions. Here are a few tips, tricks and pieces of advice I picked up there. As there were for individuals with varying pieces of advice I have grouped them into a few sections. We will begin this two part blog by discussing some of the most challenging aspects of remote work. Making a separate working environment and finding how you are most productive. 
+
+__Environment__
+
+Find a place where you can focus. While finding the perfect environment in which to focus is different for everyone. It should be a place without distractions where you are comfortable and able to focus. Having a separate work environment is also key to determining work hours. As it is difficult being a remote worker and not having to physically go into an office each day it is important to establish work space. Make a distinction in your schedule when you will be working and be in your work area during those hours.
```
Comment (laurawhalin) [inline _drafts/2015-10-23-ls-remote-working.md]: I'd change the period after "different for everyone" to a comma.

File: _drafts/2015-10-23-ls-remote-working.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+<img style="margin:auuto; padding-left: 20px" src="{{ site.url }}/assets/images/goal.png">
+
+ In particular I think that __measuring success__ is the most personal of the three points, really make this part your own. For example Anthony pays attention to his work life balance in reguards to measuring his success. Another example could be how engaged were you with your team? Were you up to date with your emails, slack messages and were active within your team discussions. Productivity is flagged by your own standards when working remote. Another point that deserves comment is __Check your metrics__. Being aware of how your work time is distributed. [Rescue Time](https://www.rescuetime.com) was suggested as a tool to track where your time is spent. This app measures and tracks your activity throughout the day.
```
Comment (laurawhalin) [inline _drafts/2015-10-23-ls-remote-working.md]: "reguards" should be "regards"
capitalize Slack

File: _drafts/2015-10-23-ls-remote-working.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+A question that came up after Anthony finished his portion of the talk was, 
+_“Do you find yourself working more or less when you work from home?”_.
+ I believe this is wonderful example of finding balance of life and work when working from home. Three of the four said they work more while one person reported working the same amount. Productivity is measured by the results we deliver and at times it is easy to become an at home workalholic. While it is extremely rewarding to be productive tracking your time spent is crucial to maintain a work life balance. The separation of space as well as time will aid you in creating this balance.  
+
+While the advice varied from speaker to speaker, when it comes to productivity and environment, do what works best for you. If you are a task oriented person who enjoys checking off their daily todo’s, add that to your work day. If you are someone who loses their entire day to work, try implementing a timer or establish a separate you work area. __Be mindful of your habits__. It is up to you to establish your goals and boundaries to set yourself up for success.
```
Comment (laurawhalin) [inline _drafts/2015-10-23-ls-remote-working.md]: "separate you work area"  I think remove "you"?

**Comment by laurawhalin** [comment]
Hey everyone, we lost the develop branch somewhere along the way, so that's why you see the last several blogs in this PR. Just roll with it ;-)
@Lydias303 nice post! Great tips and nice work on the infograph! :+1:

**Pull request merged**